### ISSUE TITLE
Fails at scale

### ISSUE THREAD

**Issue by Hornswaggler** [markheath/durable-functions-video-processor-ffmpeg#2]
The temp directory where the files are being processed is arbitrarily cleared when running at scale such that files currently being processed are arbitrarily deleted before processing is completed. Azure batch or another service with dedicated hard drive space may be required to solve the issue.

**Comment by markheath** [comment]
Yes, this is not a "production-ready" demo. I wanted to keep this demo cheap to run which is why it just uses Azure Functions disk space. But for a real world scenario, I'd probably do the processing on dedicated VMs, or perhaps Azure Container Instances with a mounted file share
